applications/visualizer/frontend/src/models/workspace.ts

```diff
@@ -106,16 +106,24 @@ export class Workspace {
     const updatedWithNeurons = await this._getAvailableNeurons(updated);
     this.updateContext(updatedWithNeurons);
   }
+  addSelectedNeuron(neuronId: string): Workspace {
```

Comment:
I think this method and `removeSelectedNeuron` needs to be removed. In a broader view `selectedNeurons` needs to be removed, we are not going to use this property anymore, it has to be replaces by the context of the synchronizer orchestrator in the 2D Graph Viewer.